### ISSUE TITLE
Error while trying to create a cluster with AWS Asia Pacific (Hong Kong) Region 'ap-east-1'

### ISSUE THREAD

**Issue by alphawelab** [rancher/rancher#20474]
**What kind of request is this (question/bug/enhancement/feature request):**
feature request

**Steps to reproduce (least amount of steps as possible):**
I am provisioning rke-cluster in `ap-east-1` (the new aws-hk-region open in 25-Apr-2019)
```
resource "rancher2_cloud_credential" "foo" {
  name = "foo"
  description = "foo test"
  amazonec2_credential_config {
    access_key = "<AWS_ACCESS_KEY>"
    secret_key = "<AWS_SECRET_KEY>"
  }
}
resource "rancher2_node_template" "foo" {
  name = "foo"
  description = "foo test"
  cloud_credential_id = "${rancher2_cloud_credential.foo.id}"
  amazonec2_config {
    ami =  "<AMI_ID>"
    region = "ap-east-1"
    security_group = ["<AWS_SECURITY_GROUP>"]
    subnet_id = "<SUBNET_ID>"
    vpc_id = "<VPC_ID>"
    zone = "<ZONE>"
  }
}
```
**Result:**
```
Error setting machine configuration from flags provided: Invalid region specified; Timeout waiting for ssh key
```
**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
Rancher 2.2.3
- Installation option (single install/HA):
HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`):

```
(paste the output here)
```

- Docker version (use `docker version`):

```
(paste the output here)
```

**Comment by loganhz** [comment]
@alphawelab This issue will be fixed and released in the next release 2.2.4.  
Please note the current default instance type for AWS node template is `t2.medium`
But `t2.medium` is not available in HK region. Please select `t3.medium` manually when you create the template.

Thanks

**Comment by rootwuj** [comment]
Tested with rancher/rancher:v2.2.4-rc19 from May 30

Steps:
1. Create an AWS node template
2. Region is ap-east-1
3. Instance type is t3.medium
4. Create EC2 cluster for region ap-east-1
5. Modify the count of cluster nodes
6. Delete cluster

Result:
Node template create successfully.
Cluster create/modify/delete successfully.

Verified fixed.
